### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by fedgut**
Requesting code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            </a>
+            <a href="#" class="aside-links">
+                <img src="assets/promo_supplier_responsibility.jpg" alt="Apple promotional img" class="figure-images">
+            </a>
+        </aside>
+    </main>
+    <footer id="second-links">
+        <div class="bottom-links"> 
+            <p> 
+            Shop the <a href="#"> Apple Online Store </a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>, or find a <a href="#">reseller </a>.
```
Comment (samgaco) [inline index.html]: Please, make the footer look closer to the original.

### ORIGINAL
original



### YOUR PROJECT
current

Comment (fedgut) [reply index.html]: Oh, for some reason I had errors when pushing, resulting in  not up to date files in the remote repo, its fixed now.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                    iPod</a>
+                <a class="nav-links" href="#">
+                    iPhone</a>
+                <a class="nav-links" href="#">
+                    iPad</a>
+                <a class="nav-links" href="#">
+                    iTunes</a>
+                <a class="nav-links" href="#">
+                    Support</a>
+                <div class="nav-search">
```
Comment (samgaco) [inline index.html]: **[OPTIONAL]** Add a [search icon](https://fontawesome.com/icons/search?style=solid) to the [search field.](https://www.w3schools.com/howto/howto_css_searchbar.asp).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+
+# Built With
+- HTML
+- CSS
+
+# Author
+- [Eduardo Gutierrez](https://github.com/fedgut)
+
+# Acknowledgments
+- [Microverse](https://microverse.org)
```
Comment (samgaco) [inline README.md]: Work on improving the readme file, so you start getting used to giving a good professional image to your repositories. You have a[ free course here ](https://www.udacity.com/course/writing-readmes--ud777
)and a good template to find inspiration [here](https://github.com/othneildrew/Best-README-Template).

**Pull request merged**